openedx/frontend-app-course-authoring
# [FC-0049] feat: Other Tags section added on tags drawer

## Description

This adds the new "Other tags" Section to tags drawer that contains the taxonomies/tags that the user doesn't have permission to see/edit. It allow to delete that tags.

## Supporting information

- Github issue: https://github.com/openedx/modular-learning/issues/204
- Internal ticket: [FAL-3720](https://tasks.opencraft.com/browse/FAL-3720)
- Depends on:
    - https://github.com/openedx/edx-platform/pull/34728
    - https://github.com/openedx/openedx-learning/pull/188

## Testing instructions

- Use this branch of `edx-platform`: https://github.com/openedx/edx-platform/pull/34728
- Run `make requirements` on cms-shell.
- Run the script on https://github.com/open-craft/taxonomy-sample-data
- Go to [taxonomy list page](http://localhost:2001/taxonomies) and import an small taxonomy: example: [template (3).csv](https://github.com/openedx/frontend-app-course-authoring/files/15241446/template.3.csv)
- Ensure that the new taxonomy doesn't have associated organizations
- Go to Studio and create a new course usign `SampleTaxonomyOrg1` as organization
- Import a course with taxonomies using the new taxonomy. You can use this example, but yu need to update the export_id of the new taxonomy: [course.phoprzm8.tar.gz](https://github.com/openedx/frontend-app-course-authoring/files/15241490/course.phoprzm8.tar.gz)
- Open the manage tags of the course or any Unit
- Check the `Other tags` section
- Click on Edit mode
- Verify that you can't add more tags in `Other tags` section.
- Delete some tags and save. Verify that the tags and count are updated.
- Delete all `Other tags` and Save. Verify that the `Other tags` section is gone.

## Other Information

TODO: It's missing to add examples to test this on https://github.com/open-craft/taxonomy-sample-data
